helm/01-check-nginx/assignment.md

```diff
@@ -0,0 +1,28 @@
+---
+slug: check-nginx
+id: o5uzceb0lsbs
+type: challenge
+title: Check that the NGINX chart is installed
+teaser: A setup script has used helm to install an NGINX chart. Verify that it worked!
+notes:
+- type: text
+  contents: Helm is one of the most popular package managers for kubernetes!
+tabs:
+- title: Shell
+  type: terminal
+  hostname: kubernetes-vm
+difficulty: basic
+timelimit: 600
+---
+Description
+===========
+
+### Describe the challenge
+
+Helm has been installed in this machine and the nginx chart as well. You can check that nginx is available with this command
+
+```
+helm list
+```
+
+If you see nginx listed, you can click on check!
```

Comment:
I think this still needs to have the copy I sent you yesterday :-) Feel free to improve it of course, but it seems like it's missing. Let me know if you cannot find it.

Reply:
I just missed a commit, check again. The description is much better now (I hope!)